common/app/views/mainAMP.scala.html

```diff
@@ -50,32 +50,33 @@
 
         <div class="main-body">
 
-            <header class="main-header @if(page.metadata.id == "film/2016/feb/02/alan-rickman-david-bowie-tributes-2016-berlin-film-festival") {variant-kicker}">
-                <a href="@LinkTo{/}" class="logo-wrapper" name="top">
-                    @fragments.inlineSvg("guardian-logo-160", "logo")
-                </a>
-
-                @* this is for testing only *@
-                @if(page.metadata.id == "film/2016/feb/02/alan-rickman-david-bowie-tributes-2016-berlin-film-festival") {
-                    <div class="main-navigation variant-kicker">
-                        <div class="main-navigation_link">
-                            <a href="http://www.theguardian.com">news</a>
-                        </div>
-                        <div class="main-navigation_link">
-                            <a href="http://www.theguardian.com/commentisfree">opinion</a>
-                        </div>
-                        <div class="main-navigation_link">
-                            <a href="http://www.theguardian.com/sport">sports</a>
-                        </div>
-                        <div class="main-navigation_link">
-                            <a href="http://www.theguardian.com/culture">arts</a>
-                        </div>
-                        <div class="main-navigation_link">
-                            <a href="http://www.theguardian.com/lifeandstyle">life</a>
-                        </div>
-                    </div>
-                }
-            </header>
+            @* this is for testing only *@
+            @defining("""bowie""".r) { hasHeader =>
```

Comment:
Echoing @JustinPinner 's point about the capital B!

__For the Why:__ 
this is for guerrilla testing in the amp demo which only crawls a small section of our articles. We have found that Bowie related articles are one of the things that exists. Therefore this can be tested with all bowie articles to help mimic an "organic" search.

Do you have a preferred search term you would rather us use? :wink: